FilmsApp/View/TabBarVC/TabBarController.swift

```diff
@@ -0,0 +1,31 @@
+import Foundation
+import UIKit
+
+class TabBarController: UITabBarController {
+    
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        let dataTransferService = DefaultDataTransferService(config: NetworkConfig(server:  Server(scheme: .https, host: "imdb-api.com")))
+        let endpoints = DefaultMoviesEnpdoints()
+        let homeVC = MainViewController(presenter: MainPresenter(dataTransferService: dataTransferService, endpoints: endpoints))
+        let favouriteVC = FavouriteController(presenter: FavouritePresenter())
+        
+        homeVC.title = "Home"
+        favouriteVC.title = "Favourite"
+        
+        let firstVC = UINavigationController(rootViewController: homeVC)
+        let secondVC = UINavigationController(rootViewController: favouriteVC)
+        
+        self.setViewControllers([firstVC, secondVC], animated: false)
+        
+        guard let items = self.tabBar.items else { return }
+        
+        let images = ["house", "star"]
+        
+        for x in 0...1 {
+            items[x].image = UIImage(systemName: images[x])
+        }
+        
+        self.tabBar.tintColor = .black
```

Comment:
This configuration must be done in "SceneDelegate". Also, when creating adding controllers to the tabBarController, firstly you need to create UITabBarItem and then assign it to each view controller that you want to add to the tabBarController, after all the viewController are configured, use "setViewControllers" in UITabBarViewController